mmv1/templates/terraform/post_import/scc_v2_folder_big_query_export_config.go.erb

```diff
@@ -0,0 +1,12 @@
+idParts := strings.Split(d.Id(), "/")
```

Comment:
I don't think that these are necessary- [in the generated code here](https://github.com/modular-magician/terraform-provider-google/compare/auto-pr-11517-old..auto-pr-11517#diff-fe84af1c37d99a7e8a6239838714959c567beb055437097be30ca13a79496a09R424-R429) string formats from `import_format` in FolderSccBigQueryExports.yaml are passed to the tpgresource.ParseImportId function. This  function both [validates the user-provided string](https://github.com/hashicorp/terraform-provider-google/blob/6c95ec69674287761ecf5ddb43e07eeb778f574d/google/tpgresource/import.go#L77) to make sure it's a correct format, and [pulls out sections of the string and sets them as values for the fields of the resource](https://github.com/hashicorp/terraform-provider-google/blob/6c95ec69674287761ecf5ddb43e07eeb778f574d/google/tpgresource/import.go#L49). I think it currently does everything in this post_import code

Reply:
Okay thank you, will remove it. I have started adding it because of the comment that you made on this PR [11346](https://github.com/GoogleCloudPlatform/magic-modules/pull/11346), so I have made a practice of adding it. Can you let me know in which cases do we need to add the post_import and in which case we don't. Thanks.

Reply:
Sorry, I was incorrect to give that feedback last time. I've had to debug some import failures recently so my understanding has improved since then.

Times where a post import might be beneficial are when a resource has the [import format specified as just {{name}} like here](https://github.com/GoogleCloudPlatform/magic-modules/blob/d87891e2492f2dcc5a666e6e99e4917eab882761/mmv1/products/datacatalog/PolicyTag.yaml#L33). In that case the name field will be set to the user provided string but its constituent parts (ie project, location) won't be set. In this case though it'd be best to update the import format. Also if there virtual fields a developer might want to set their values in the import step.